app/_gateway_entities/vault.md

```diff
@@ -0,0 +1,146 @@
+| [Azure Key Vaults](/how-to/configure-azure-key-vaults-as-a-vault-backend) | <img src="/app/assets/icons/close.svg" alt="Cross icon"> | <img src="/app/assets/icons/check.svg" alt="Check icon">   | <img src="/app/assets/icons/check.svg" alt="Check icon">   | <img src="/app/assets/icons/check.svg" alt="Check icon">        |
+| [Google Cloud Secret](/how-to/configure-google-cloud-secret-as-a-vault-backend) | <img src="/app/assets/icons/close.svg" alt="Cross icon"> | <img src="/app/assets/icons/check.svg" alt="Check icon">   | <img src="/app/assets/icons/check.svg" alt="Check icon">   | <img src="/app/assets/icons/check.svg" alt="Check icon">        |
+| [Hashicorp Vault](/how-to/configure-hashicorp-vault-as-a-vault-backend) | <img src="/app/assets/icons/close.svg" alt="Cross icon"> | <img src="/app/assets/icons/check.svg" alt="Check icon">   | <img src="/app/assets/icons/check.svg" alt="Check icon">   | <img src="/app/assets/icons/check.svg" alt="Check icon">        |
+
+
+## How do I use secrets stored in a Vault?
+
+When you want to use a secret stored in a Vault, you can reference the secret with a `vault` reference. You can use the `vault` reference in places such as `kong.conf`, declarative configuration files, logs, or in the UI.
+
+A secret reference points to a string value. No other data types are currently supported.
+
+The Vault backend may store multiple related secrets inside an object, but the reference
+should always point to a key that resolves to a string value. For example, the following reference:
+
+```
+{vault://hcv/pg/username}
+```
+
+Would point to a secret object called `pg` inside a HashiCorp Vault, which may return the following value:
+
+```json
+{
+  "username": "john",
+  "password": "doe"
+}
+```
+
+<!-- vale off -->
+{{site.base_gateway}} receives the payload and extracts the `"username"` value of `"john"` for the secret reference of
+`{vault://hcv/pg/username}`.
+<!-- vale on -->
+
+## How do I manage secrets stored in a Vault?
+
+Although {{site.base_gateway}} can't create or add secrets to a Vault, it can help you [manage your secrets](/secrets-management/). By default, {{site.base_gateway}} automatically rotates secrets *once every minute* in the background. You can also further configure how often {{site.base_gateway}} rotates secrets using the Vault entity configuration and if it rotates secrets on failure (such as a database authentication failure).
+
+For more information, see [Secret rotation](/gateway/secrets-management/secret-rotation/).
+
+## Schema
+
+The Vault entity can only be used once the database is initialized. Secrets for values that are used before the database is initialized can’t make use of the Vaults entity.
+
+{% entity_schema %}
```

Comment:
The schema won't render, any idea why?

Reply:
the schema does render for me both  locally and in the preview app. do you have `VITE_PORTAL_API_URL` set in an `.env` file?